jaeger/charts/linkerd-jaeger/templates/namespace-metadata-rbac.yaml

```diff
@@ -13,7 +14,9 @@ metadata:
   name: namespace-metadata
   namespace: {{.Release.Namespace}}
 {{- include "partials.image-pull-secrets" .Values.imagePullSecrets }}
+{{-end}}
 ---
+{{- if not .Values.skipNamespaceMetadataJobs}}
```

Comment:
It's more practical to wrap the entire file in this guard, instead of each resource.

Reply:
thanks for the suggestion. I will make changes to wrap the entire file.